lib/custom-validator-logger.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+module.exports = function customValidatorLogger (ms, cmd, err, args) {
```

Comment:
Maybe `customValidatorLogFormatter`? To me, `customValidatorLogger` makes me think it's going to actually log.